tools/sdk-ai-bots/azure-sdk-qa-bot-backend/test/utils_test.go

```diff
@@ -0,0 +1,58 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/azure-sdk-tools/tools/sdk-ai-bots/azure-sdk-qa-bot-backend/model"
+	"github.com/azure-sdk-tools/tools/sdk-ai-bots/azure-sdk-qa-bot-backend/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFilterInvalidReferenceLinks_ValidLinks(t *testing.T) {
+	chunks := []string{
+		"Check out this guide: https://example.com/path?param=value&other=123#section",
```

Comment:
Are those chunks returned by the AI search? And are they document snippets? I’d like to know whether the links point to the documents containing those snippets, or if they are links found within the documents